roles/pulp_common/README.md

```diff
@@ -152,6 +154,13 @@ If it is set to "packages", the following variables are used, or behave *differe
   installer will fix it.)
   Defaults to `false`.
 
+Role Variables for advanced usage
+---------------------------------
+
+* `pulp_version`: Specify a specific version of pulpcore one would like to install or upgrade to.
```

Comment:
Maybe we should call it `pulpcore_version`.
We should be able to rename it today, since it was never advertised before.

Reply:
This would be a backward incompatible change. `pulp_version` is already present before this PR and hence user might be leveraging it. While I agree it would make more sense, (maybe going to a deprecation period) - I don't think this should happen as part of this PR.

Reply:
Changing things that are not part of an official interface is the privilege of upstream.
This variable was even described as "intentionally not advertised".

Reply:
@mikedep333 @fao89 @dkliban what are your thoughts on that one ?

Reply:
Yeah, I am in favor of renaming it to `pulpcore_version`.